plugins/rbac-backend/README.md

```diff
@@ -111,6 +111,14 @@ g, user:default/bob, role:default/team_a
 g, group:default/team_b, role:default/team_b
 ```
 
+---
+
+**NOTE**
+
+When adding a role in the permission policies configuration file it is adviced to associate the role with atleast one permission policy with effect `allow`
```

Comment:
`Admin user, ensure that they have read permission for `catalog-entity`` is needed for rbac UI plugin, because the create form requires catalog entity read permission to fetch the users and groups